server/plugin.go

```diff
@@ -427,24 +396,22 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 
 	post, appErr := p.API.CreatePost(post)
 	if appErr != nil {
-		mlog.Error("Error creating a new post with the new meeting: " + appErr.Error())
-		http.Error(w, appErr.Error(), http.StatusInternalServerError)
-		return
+		return errors.Wrapf(appError, "cannot create meeting. An error occured while creating a post with a meeting"),
+			http.StatusInternalServerError
 	}
 
 	appErr = p.API.KVSet(fmt.Sprintf("%v%v", PostMeetingKey, newMeetingResponse.MeetingID), []byte(post.Id))
 	if appErr != nil {
-		mlog.Error("Error writing meeting id to the database: " + appErr.Error())
-		http.Error(w, appErr.Error(), http.StatusInternalServerError)
-		return
+		return errors.Wrapf(appError, "cannot create meeting. An error occured while saving the meeting ID in the database"),
+			http.StatusInternalServerError
 	}
 
 	if err := json.NewEncoder(w).Encode(&newMeetingResponse); err != nil {
-		mlog.Error("Error encoding the new meeting response: " + err.Error())
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return errors.Wrapf(err, "cannot create meeting. An error occured while encoding the new meeting response"),
+			http.StatusInternalServerError
 	}
 
-	return
+	return nil, http.StatusOK
 }
 
 func (p *Plugin) handleProfileImage(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why this does not return error and int?

Reply:
My mistake. I've just changed it to return error and int.